package.json

```diff
@@ -50,7 +50,7 @@
     }
   },
   "dependencies": {
-    "sqlite3": "4.0.0"
+    "sqlite3": "4.0.2"
```

Comment:
How about `"sqlite3": "^4.0.0"`? Then people will be able to use version they want...

Reply:
4.0.0 and 4.0.1 have a flaw which prevents it from downloading the precompiled binary behind proxies. There's no practical reason to stay at 4.0.0 or 4.0.1. At least with 4.0.2 mentionned explicitely you're sure people who will only run `npm install sqlite` will fetch the correct version.